src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/PerfTests/DebuggerSteppingTestcase.xml

```diff
@@ -1,15 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <TaoTest xmlns="http://microsoft.com/schemas/VSLanguages/TAO">
   <InitTest>
-    <StartTarget />
+    <StartTarget DontResetOptions="true" />
```

Comment:
Seems harmless, but is there a reason we're doing this?  Are other perf tests doing it?